### PR TITLE
Link to charter

### PR DESCRIPTION
**Pull request by carolynvs**
This links to our working charter. Once it is merged, I'll update this to point to the the permanent location.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,8 +60,8 @@ Also see the [developer's guide](./docs/devguide.md) for information on how to
 build and test the code.
 
 We have weekly meetings - see
-[Kubernetes SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md)
-(search for "Service Catalog") for the exact date and time. For meeting agendas
+[our SIG Readme](https://github.com/kubernetes/community/blob/master/sig-service-catalog/README.md#meetings)
+for details. For meeting agendas
 and notes, see [Kubernetes SIG Service Catalog Agenda](https://docs.google.com/document/d/17xlpkoEbPR5M6P5VDzNx17q6-IPFxKyebEekCGYiIKM/edit).
```
Comment (MHBauer) [inline README.md]: comment while we're editing here. we have about a year of meetings in our current doc, should we consider rotating to a new doc?

**Comment by duglin** [comment]
Yea but just move to the end of the existing archive doc.

File: README.md (hunk 1 above)
Comment (carolynvs) [reply README.md]: Does having a large doc cause any problems?

Comment (duglin) [reply README.md]: gdocs doesn't load large docs well sometimes, and based on the device you're on.
Now that I'm back at my computer I'll move stuff....

Comment (carolynvs) [reply README.md]: 👍 Thanks!

**Pull request merged**